scripts/process_cams_to_csv.py

```
@@ -0,0 +1,253 @@
+        print("="*60 + "\n")
+        
+        return df
+        
+    except Exception as e:
+        logger.error(f"Error processing {nc_path}: {e}")
+        import traceback
+        traceback.print_exc()
+        return pd.DataFrame()
+
+
+def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)